hvac/api/auth_methods/userpass.py

```diff
@@ -4,14 +4,15 @@
 from hvac.api.vault_api_base import VaultApiBase
 
 DEFAULT_MOUNT_POINT = 'userpass'
+DEFAULT_POLICIES = 'default'
 
 
 class Userpass(VaultApiBase):
     """USERPASS Auth Method (API).
     Reference: https://www.vaultproject.io/api/auth/userpass/index.html
     """
 
-    def create_or_update_user(self, username, password, mount_point=DEFAULT_MOUNT_POINT):
+    def create_or_update_user(self, username, password, policies=DEFAULT_POLICIES, mount_point=DEFAULT_MOUNT_POINT):
```

Comment:
We've found that specifying this sort of default argument and always passing it along in the API request leads to unexpected issues generally. How would you feel about instead defaulting to `None` and taking advantage of the recently added `remove_nones()` / https://github.com/hvac/hvac/blob/develop/hvac/utils.py#L289 in the subsequent params deal a la:

```python
        params = utils.remove_nones({
            'policies': policies,
            'password': password,
        })
```

Reply:
Updated